Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.68 AS builder
```

Comment:
Are you basing this version off `rust-toolchain.toml`? For this commit it's still set to `1.68.2`

Reply:
Yes. Since their `main` and `devnet` branches have already moved to `1.70`, it'll be a matter of time when it hits the other branches. Since we don't have multiple branches, we'll just use the latest for all of them, which should hopefully be compatible anyway.